### PR TITLE
Sv 7766/chameleon modal

### PR DESCRIPTION
**Pull request by mandymichel**
To test: view the chameleon modal in storybook and ensure that it meets the specfications in here: https://app.zeplin.io/project/5ec586c8721ad446af909988/screen/5ec586fe85a6b34883c3671b and will be useable for this page (see Teams for comps)
Please give feedback- not sure if I made this just flexible enough or too flexible!

### CODE REVIEW

File: packages/modal/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@chameleon-ds/modal",
+  "version": "1.1.5",
```
Comment (ryuhhnn) [inline packages/modal/package.json:3]: Can you set the version to 0.1.0?

File: packages/modal/src/chameleon-modal.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import { LitElement, TemplateResult, html, property } from "lit-element";
+import { nothing, svg, SVGTemplateResult } from "lit-html";
+import { classMap } from "lit-html/directives/class-map";
+import style from "./chameleon-modal-style";
+import "@chameleon-ds/card";
+import "@chameleon-ds/button";
+
+export default class Chameleonmodal extends LitElement {
```
Comment (ryuhhnn) [inline packages/modal/src/chameleon-modal.ts:8]: Can you camelcase "ChameleonModal"?

File: packages/modal/src/chameleon-modal.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  /**
+   * Properties
+   */
+  @property({ type: Boolean, reflect: true })
+  open = false;
+
+  // Card has a close icon
+  @property({ type: Boolean, reflect: true })
+  "dismissible" = false;
```
Comment (ryuhhnn) [inline packages/modal/src/chameleon-modal.ts:17]: Minor nitpick, can you remove the quotes around "dismissible"?

File: packages/modal/src/chameleon-modal.ts [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+  @property({ type: Boolean, reflect: true })
+  "fullScreen" = false;
```
Comment (ryuhhnn) [inline packages/modal/src/chameleon-modal.ts:20]: same here, can you remove the quotes?

File: packages/modal/src/chameleon-modal.ts [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+   * Styles
+   */
+  static styles = [style];
+
+  /**
+   * Template
+   */
+  render(): TemplateResult {
+    return html`
+      <div
```
Comment (ryuhhnn) [inline packages/modal/src/chameleon-modal.ts:32]: This div doesn't have a closing tag

File: packages/modal/src/chameleon-modal.ts [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    const closeIcon = slots.find((slot) => slot.slot === "close-icon");
+
+    if (closeIcon === undefined)
+      return svg`<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-x"><line x1="18" y1="6" x2="6" y2="18"></line><line x1="6" y1="6" x2="18" y2="18"></line></svg>`;
+    else return html` <slot name="close-icon"></slot> `;
+  }
+}
+
+if (!window.customElements.get("chameleon-modal"))
+  window.customElements.define("chameleon-modal", Chameleonmodal);
```
Comment (ryuhhnn) [inline packages/modal/src/chameleon-modal.ts:90]: Remember to camelcase here too

File: packages/skeleton/__tests__/chameleon-skeleton.test.ts [hunk 7]
```diff
@@ -10,6 +10,7 @@ describe("chameleon-skeleton", () => {
     const el = await fixture(html` <chameleon-skeleton></chameleon-skeleton> `);
     el.theme = "secondary";
     el.cHeight = 50;
+    console.log(el);
```
Comment (ryuhhnn) [inline packages/skeleton/__tests__/chameleon-skeleton.test.ts:13]: Can you remove this console log?

File: packages/modal/src/chameleon-modal.ts [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+                    icon-only
+                    @click="${this._togglemodal}"
+                  >
+                    ${this.closeIcon}
+                  </chameleon-button>
+                `
+              : nothing
+          }
+        </div>
+        <hr/>
```
Comment (ryuhhnn) [inline packages/modal/src/chameleon-modal.ts:57]: Instead of using a horizontal rule, can we remove this and just add a border bottom to the `div.header`?

Comment (mandymichel) [reply packages/modal/src/chameleon-modal.ts:57]: Good idea!

**Pull request merged**